jenkinsfiles/RHEL_Check.groovy

```diff
@@ -193,7 +194,32 @@ pipeline {
                     }
                 }
 
+                stage('Check_code_changes') {
+                    steps {
+                        script {
+                            dir("./src") {
+                                def code_changes = sh(
+                                    script: """
+                                            DOC_STRINGS = $(git diff --name-only main |grep ^documentation)
+                                            STRINGS = $(git diff --name-only origin/main)
+                                            if [[ DOC_STRINGS != STRINGS ]]; then
```

Comment:
sh is a simple POSIX shell and that doesn't support the [[ construct, which is a bash thing. Also you should use $ to get value from variable to compare
```suggestion
                                            if [ ${DOC_STRINGS} != ${STRINGS} ]; then
```

Reply:
Fixed